pages/common/ocamlc.md

```diff
@@ -0,0 +1,12 @@
+# ocamlc
+
+> The OCaml bytecode compiler.
+> Produces executables runnable by OCaml interpreter.
+
+- Create named executable from source file:
+
+`ocamlc -o {{path/to/binary}} {{path/to/source_file.ml}}`
+
+- Create automatically named executable from source file:
```

Comment:
The "automatically named executable" part is not super-clear to me. I get that the name of the executable is decided by ocamlc, but can we reword it to make it clearer ?

And also, let's change executable to binary to be consistent with the examples.

Reply:
Check it out now, it should be much more clear.

Reply:
Much better now. But I'm curious to know what is the final name of the binary that gets created ?